src/index.js

```diff
@@ -48,15 +35,9 @@ function bufToBigInt(buf) {
  * @returns
  */
 const hash = (input = "", length = bigLength) => {
-  // The salt should be long enough to be globally unique across the full
-  // length of the hash. For simplicity, we use the same length as the
-  // intended id output, defaulting to the maximum recommended size.
-  const salt = createEntropy(length);
-  const text = input + salt;
-
   // Drop the first character because it will bias the histogram
   // to the left.
-  return bufToBigInt(sha3(text)).toString(36).slice(1);
+  return bufToBigInt(sha3(input)).toString(36).slice(1);
```

Comment:
Do you still need to drop the character now that the algorithm for converting the SHA to an int is not conflating different values? In my histogram tests, I found the full result of the output to be adequately distributed.

Reply:
Yes, I do. Not sure why, but this works for now.

Reply:
Maybe it's an issue with the input, then? SHA hash functions exhibit uniformity, so there should be an equally likely chance of any given possible output. If all we're doing here is converting the output into a base36 number, there's nothing that should bias the first character in any particular direction.